src/contexts/build.context.tsx

```diff
@@ -197,6 +197,10 @@ async function deleteBuild(dispatch: Dispatch, id: string) {
   });
 }
 
+async function clearBuild(dispatch: Dispatch, id: string) {
```

Comment:
let's instead directly dispatch actions, if it doesn't contain any api calls or additional logic